src/app/views/communities/view/overview/overview.ts

```diff
@@ -165,6 +208,30 @@ export default class RouteCommunitiesViewOverview extends BaseRouteComponent {
 		);
 		this.knownMembers = overviewPayload.knownMembers || [];
 		this.knownMemberCount = overviewPayload.knownMemberCount || 0;
+
+		Meta.description = this.$gettextInterpolate(
+			// tslint:disable-next-line:max-line-length
+			'Welcome to the %{ name } community on Game Jolt! Find and explore %{ name } fan art, lets plays and catch up on the latest news and theories!',
+			{ name: this.community.name }
+		);
+
+		Meta.fb = {
+			type: 'website',
+			title: this.routeTitle,
+			description: Meta.description,
+
+			// TODO provide a nice image specifically for SEO from the backend.
+			// image: this.community.header ? this.community.header!.mediaserver_url,
```

Comment:
Why is this todo? Can't we use header or thumb?

Reply:
Header lacks context and is used seemingly for decor. Thumb is too small.
I think we should use an image that conveys what this is about, and is complementary to the text we're providing.

Reply:
We are planning to add a thumbnail similar to games iirc, maybe we can use that once we have it.